GiniVision/Classes/HelpMenuViewController.swift

```diff
@@ -28,7 +43,7 @@ final public class HelpMenuViewController: UITableViewController {
                                comment: "help menu third item text"), 3)
         ]
         
-        if GiniConfiguration.sharedConfiguration.openWithEnabled {
+        if self.giniConfiguration.openWithEnabled {
```

Comment:
Why didn't you omit the `self`?

Reply:
`lazy` variables need it, otherwise it doesn't compile.